### PR TITLE
Pai/New Job Profiles

### PR DESCRIPTION
**Pull request by Winterman981**
## Summarize what is being added
Added 8 more job profiles along with their respective grading criteria.

## Please describe how to test
Wait for implementation.

## Relevant Screenshots
image
image
image

## What Week of the 603 cycle is this due in?
Week 3

### CODE REVIEW

File: ApplyToTheIndustry/Assets/ScriptableObjects/Grading Profiles/UIUXMichaelSoft.asset [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      name: Programming
+      value: 0
+      skillType: 0
+    design:
+      name: Design
+      value: 45
+      skillType: 1
+    graphic_art:
+      name: Graphics and Art
+      value: 0
```
Comment (amonjerro) [inline ApplyToTheIndustry/Assets/ScriptableObjects/Grading Profiles/UIUXMichaelSoft.asset]: UI/UX should probably have some knowledge of graphic design

File: ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/LevelDesignerLatus.asset [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  company: {fileID: 11400000, guid: d735f67bc9e8f0c488ca3389c6702cde, type: 2}
+  positionName: Level Designer
+  jobSummaryLines:
+  - Our team is looking for a level designer for our game.
+  - 'Expectations:'
+  jobReqLines:
+  - '>Create a believable space that fits in the context of gameplay'
+  - '>Ability to evaluate and apply feedbacks'
+  - '>Keen eye for details'
+  - '>Some programming skills in C++, C#, or Java'
```
Comment (amonjerro) [inline ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/LevelDesignerLatus.asset]: We're trying to move away from named languages - could you rephrase this?

File: ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/LevelDesignerGauge.asset [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  company: {fileID: 11400000, guid: f7d62fe3dc4190e4d92cb57728dfc5b7, type: 2}
+  positionName: Junior Level Designer
+  jobSummaryLines:
+  - Our company is looking for junior level designers.
+  - 'Requirements:'
+  jobReqLines:
+  - '>Strong understanding of level design in video games'
+  - '>Passion for video game development'
+  - '>Experienced working in game development process'
+  - '>Familarity with scrum meeting and workflow'
```
Comment (amonjerro) [inline ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/LevelDesignerGauge.asset]: Scrum is a capitalized word, also should say "meetings", plural

File: ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/QuestDesignerLariat.asset [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  m_Script: {fileID: 11500000, guid: d8cfffae36be18b40b59e8e84d47a6ad, type: 3}
+  m_Name: QuestDesignerLariat
+  m_EditorClassIdentifier: 
+  company: {fileID: 11400000, guid: 10299a39b72aab747a9b16ceee7f8742, type: 2}
+  positionName: Quest Narrative Designer
+  jobSummaryLines:
+  - We are looking for a passionate narrative designer!
+  - 'Requirements for this position:'
+  jobReqLines:
+  - '>Passion in video games'' narrative and storytelling'
```
Comment (amonjerro) [inline ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/QuestDesignerLariat.asset]: I would rephrase this as "Passion for narrative and storytelling in videogames".

File: ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/UIUXMichaelsoft.asset [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  m_Name: UIUXMichaelsoft
+  m_EditorClassIdentifier: 
+  company: {fileID: 11400000, guid: d596ba1022aed3641984cad46486500b, type: 2}
+  positionName: UI/UX Designer
+  jobSummaryLines:
+  - We are looking for an individual with experience in UI/UX design.
+  - 'Requirements and qualifications:'
+  jobReqLines:
+  - '>Knowledge of equitable and usable UI design'
+  - '>Ability to brainstorm and formulate concepts for rapid-prototyping'
```
Comment (amonjerro) [inline ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/UIUXMichaelsoft.asset]: rapid prototyping is not a hyphenate

**Review by amonjerro — COMMENTED** [review]
Overall really good! Some minor things to correct but you did a good job Pai!

File: ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/GameProducerQuad.asset [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+  m_Script: {fileID: 11500000, guid: d8cfffae36be18b40b59e8e84d47a6ad, type: 3}
+  m_Name: GameProducerQuad
+  m_EditorClassIdentifier: 
+  company: {fileID: 11400000, guid: 9cfc23b03c89faa4fa9248e30f7d815a, type: 2}
+  positionName: Game Producer
+  jobSummaryLines:
+  - We are urgently looking for a game producer!
+  - 'Skills required:'
+  jobReqLines:
+  - '>5-10 years of working in the gaming industry'
```
Comment (RitoxicGame) [inline ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/GameProducerQuad.asset]: "years of working" -> "years experience working"
"gaming industry" -> "games industry"
(**gaming** industry refers primarily to gambling, casinos, etc.)

File: ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/GameProducerQuad.asset [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+  - '>In-depth knowledge of game development process'
```
Comment (RitoxicGame) [inline ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/GameProducerQuad.asset]: process**es**

File: ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/GameProducerQuad.asset [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+  - '>Efficient skills in problem-solving and arranging schedules'
+  - '>Ability to work and communicate with other team leads'
+  - '>Ability to receive and apply feedbacks critically'
```
Comment (RitoxicGame) [inline ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/GameProducerQuad.asset]: **feedback** (singular)

File: ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/GameProducerQuad.asset [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: d8cfffae36be18b40b59e8e84d47a6ad, type: 3}
+  m_Name: GameProducerQuad
+  m_EditorClassIdentifier: 
+  company: {fileID: 11400000, guid: 9cfc23b03c89faa4fa9248e30f7d815a, type: 2}
+  positionName: Game Producer
+  jobSummaryLines:
+  - We are urgently looking for a game producer!
```
Comment (RitoxicGame) [inline ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/GameProducerQuad.asset]: Consider removing this exclamation point
It feels unnecessary

File: ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/LeadAudioSleazy.asset [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+  m_Script: {fileID: 11500000, guid: d8cfffae36be18b40b59e8e84d47a6ad, type: 3}
+  m_Name: LeadAudioSleazy
+  m_EditorClassIdentifier: 
+  company: {fileID: 11400000, guid: 9acb2686d0190004ba5eb76f1e26f275, type: 2}
+  positionName: Lead Audio Designer
+  jobSummaryLines:
+  - Our team is looking for a lead sound designer.
+  - 'Requirements and expectations:'
+  jobReqLines:
+  - '>Experience working with digital workstation'
```
Comment (RitoxicGame) [inline ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/LeadAudioSleazy.asset]: recommended change:
"digital **audio** workstation**s**."

File: ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/LevelDesignerGauge.asset [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+  m_EditorClassIdentifier: 
+  company: {fileID: 11400000, guid: f7d62fe3dc4190e4d92cb57728dfc5b7, type: 2}
+  positionName: Junior Level Designer
+  jobSummaryLines:
+  - Our company is looking for junior level designers.
+  - 'Requirements:'
+  jobReqLines:
+  - '>Strong understanding of level design in video games'
+  - '>Passion for video game development'
+  - '>Experienced working in game development process'
```
Comment (RitoxicGame) [inline ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/LevelDesignerGauge.asset]: "experience**d**" -> "experience"

File: ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/LevelDesignerLatus.asset [hunk 12]
```diff
@@ -0,0 +1,26 @@
...
+  m_Name: LevelDesignerLatus
+  m_EditorClassIdentifier: 
+  company: {fileID: 11400000, guid: d735f67bc9e8f0c488ca3389c6702cde, type: 2}
+  positionName: Level Designer
+  jobSummaryLines:
+  - Our team is looking for a level designer for our game.
+  - 'Expectations:'
+  jobReqLines:
+  - '>Create a believable space that fits in the context of gameplay'
+  - '>Ability to evaluate and apply feedbacks'
```
Comment (RitoxicGame) [inline ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/LevelDesignerLatus.asset]: **feedback** (singular)

File: ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/LevelDesignerLatus.asset (hunk 2 above)
Comment (RitoxicGame) [reply ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/LevelDesignerLatus.asset]: ^

File: ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/LevelDesignerLatus.asset [hunk 13]
```diff
@@ -0,0 +1,26 @@
...
+  - '>Keen eye for details'
+  - '>Some programming skills in C++, C#, or Java'
+  - '>Knowledge of sound design is not required, but will be a plus'
```
Comment (RitoxicGame) [inline ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/LevelDesignerLatus.asset]: remove "is not required, but" (redundant)

File: ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/ProductionCoordinatorFreezerBurn.asset [hunk 14]
```diff
@@ -0,0 +1,24 @@
...
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: d8cfffae36be18b40b59e8e84d47a6ad, type: 3}
+  m_Name: ProductionCoordinatorFreezerBurn
+  m_EditorClassIdentifier: 
+  company: {fileID: 11400000, guid: d0ddb5a02bb3d9a4f86d3ef8fd732359, type: 2}
+  positionName: Production Coordinator
+  jobSummaryLines:
+  - Our company is looking for someone willing to take a lead in our production.
```
Comment (RitoxicGame) [inline ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/ProductionCoordinatorFreezerBurn.asset]: either "take **the** lead **on** our production **team**" or "take a lead**ing role on** our production **team**"

Comment (RitoxicGame) [reply ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/ProductionCoordinatorFreezerBurn.asset]: (would recommend the latter, personally

File: ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/ProductionCoordinatorFreezerBurn.asset [hunk 15]
```diff
@@ -0,0 +1,24 @@
...
+  - 'Required skills:'
+  jobReqLines:
+  - '>Ability to work in fast-paced environment under pressure'
```
Comment (RitoxicGame) [inline ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/ProductionCoordinatorFreezerBurn.asset]: recommended addition: "_**a**_ fast-paced environment"

File: ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/QuestDesignerLariat.asset (hunk 4 above)
Comment (RitoxicGame) [reply ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/QuestDesignerLariat.asset]: ^

File: ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/QuestDesignerLariat.asset [hunk 16]
```diff
@@ -0,0 +1,25 @@
...
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: d8cfffae36be18b40b59e8e84d47a6ad, type: 3}
+  m_Name: QuestDesignerLariat
+  m_EditorClassIdentifier: 
+  company: {fileID: 11400000, guid: 10299a39b72aab747a9b16ceee7f8742, type: 2}
+  positionName: Quest Narrative Designer
+  jobSummaryLines:
+  - We are looking for a passionate narrative designer!
```
Comment (RitoxicGame) [inline ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/QuestDesignerLariat.asset]: Again, recommend removing the exclamation point.

File: ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/UIUXMichaelsoft.asset [hunk 17]
```diff
@@ -0,0 +1,26 @@
...
+  m_Script: {fileID: 11500000, guid: d8cfffae36be18b40b59e8e84d47a6ad, type: 3}
+  m_Name: UIUXMichaelsoft
+  m_EditorClassIdentifier: 
+  company: {fileID: 11400000, guid: d596ba1022aed3641984cad46486500b, type: 2}
+  positionName: UI/UX Designer
+  jobSummaryLines:
+  - We are looking for an individual with experience in UI/UX design.
+  - 'Requirements and qualifications:'
+  jobReqLines:
+  - '>Knowledge of equitable and usable UI design'
```
Comment (RitoxicGame) [inline ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/UIUXMichaelsoft.asset:21]: "equitable" does not appear to make sense here.
consider removing, or alternatively replace with "clean" or somesuch

File: ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/UIUXMichaelsoft.asset (hunk 5 above)
Comment (RitoxicGame) [inline ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/UIUXMichaelsoft.asset]: "brainstorm" and "formulate concepts" feel redundant with one another

consider removing the latter

File: ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/UIUXMichaelsoft.asset [hunk 18]
```diff
@@ -0,0 +1,26 @@
...
+  - '>Ability to brainstorm and formulate concepts for rapid-prototyping'
+  - '>Ability to communicate with our internal researchers'
```
Comment (RitoxicGame) [inline ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/UIUXMichaelsoft.asset]: narrative feedback: "internal researchers" is fine, but I'd recommend phrasing it as "Ability to communicate with cross-disciplinary teams" or something like that

File: ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/UIUXMichaelsoft.asset [hunk 19]
```diff
@@ -0,0 +1,26 @@
...
+  - '>Strong social skills, being able to interview testers for desired feedbacks'
```
Comment (RitoxicGame) [inline ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/UIUXMichaelsoft.asset]: rephrase:
"Experience collecting and analyzing feedback from playtesters"

also: **feedback** (singular)

File: ApplyToTheIndustry/Assets/ScriptableObjects/Grading Profiles/UIUXMichaelSoft.asset (hunk 1 above)
Comment (Winterman981) [reply ApplyToTheIndustry/Assets/ScriptableObjects/Grading Profiles/UIUXMichaelSoft.asset]: Added graphics design score of 30 to the grading profile

File: ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/LevelDesignerLatus.asset (hunk 2 above)
Comment (Winterman981) [reply ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/LevelDesignerLatus.asset]: Changed to ">Some programming skills in commonly used coding languages"

File: ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/UIUXMichaelsoft.asset (hunk 17 above)
Comment (Winterman981) [reply ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/UIUXMichaelsoft.asset:21]: I disagree because this is a word used for UI/UX design, it means "accessibility" if I can recall

File: ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/UIUXMichaelsoft.asset (hunk 5 above)
Comment (Winterman981) [reply ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/UIUXMichaelsoft.asset]: I feel like there is a nuance between these 2 words. Brainstorm is to come up with ideas, but formulating it means making said ideas tangible. I remember seeing that on real job postings

File: ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/UIUXMichaelsoft.asset (hunk 17 above)
Comment (RitoxicGame) [reply ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/UIUXMichaelsoft.asset:21]: mmmmmmmmm
that makes sense, then

File: ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/UIUXMichaelsoft.asset (hunk 5 above)
Comment (RitoxicGame) [reply ApplyToTheIndustry/Assets/ScriptableObjects/Job Postings/UIUXMichaelsoft.asset]: understood
If this is pulled from actual job postings, then that would take priority over my suggestion

**Review by amonjerro — APPROVED** [review]
LGTM
